model/sys/vmem_utils.sail

```diff
@@ -160,7 +160,7 @@ function vmem_write_addr(vaddr, width, data, acc, aq, rl, res) = {
 
 // External API
 
-function check_misaligned(vaddr : virtaddr, width : word_width) -> bool = {
+function check_misaligned(vaddr : virtaddr, width : word_width_wide) -> bool = {
```

Comment:
This doesn't feel like quite the right type. Technically it's the same as `is_mem_width()` but it was kind of meant to be specific to instruction-encoding of access widths.

You could just use `nat1` for now since the underlying functions use that anyway (in future we should constrain them to something non-infinite for performance reasons, but it's fine for now).

```suggestion
function check_misaligned(vaddr : virtaddr, width : nat1) -> bool = {
```